components/OutivityForm/index.js

```diff
@@ -46,6 +95,7 @@ export default function NewOutivityForm({ createOutivity, handleCancel }) {
               id="outivityArea"
               placeholder="In which city/area is it located?"
               required
+              defaultValue={isEdit ? outivity.area ?? "" : null}
```

Comment:
Do you need this at this point in time? Since your `outivityArea` input is required anyway, there should never be a empty or undefined `outivity.area`. But if you want, you could also leave it in, if it's for future proofing and you need it in future versions of your app 😉
```suggestion
              defaultValue={isEdit ? outivity.area : null}
```

Reply:
Good point, you are absolutely right. As we don't need this for now, we changed all the `defaultValue`s